lib/org/chromium/webidl/WebKitParser.js

```diff
@@ -56,6 +56,17 @@ foam.CLASS({
               ExtendedAttributeLogicalOperators: tseq(
                   sym('identifier'), '=', sym('identifier'), alt('&', '|'),
                   tplus(sym('identifier'), alt('&', '|'))),
+              ExtendedAttributeFnCall: tseq(
+                sym('identifier'), '(', sym('identifier'), ')'),
```

Comment:
Whoops. This has actually been removed because the semantic rule no longer necessary (only needed for files withi `win/interfaces`